static/js/components/FollowupRequestPage.jsx

```diff
@@ -35,23 +39,55 @@ const FollowupRequestPage = () => {
   const classes = useStyles();
   const dispatch = useDispatch();
 
-  const [queryInProgress, setQueryInProgress] = useState(false);
+  const [selectedInstrumentId, setSelectedInstrumentId] = useState(null);
   const [fetchParams, setFetchParams] = useState({
     pageNumber: 1,
     numPerPage: defaultNumPerPage,
   });
 
   useEffect(() => {
-    dispatch(followupRequestActions.fetchFollowupRequests());
-  }, [dispatch]);
+    const getInstruments = async () => {
+      // Wait for the allocations to update before setting
+      // the new default form fields, so that the allocations list can
+      // update
+
+      const result = await dispatch(instrumentActions.fetchInstruments());
+
+      const { data } = result;
+      setSelectedInstrumentId(data[0]?.id);
+    };
+
+    getInstruments();
```

Comment:
This is async, so doesn't it require an `await`?

Reply:
I am not sure, the await is in the function?

Reply:
I think it's fine in this case; you just want to make sure the dispatch eventually happens, and it will once the getInstruments function has been called.

`async` functions returns promises, so if you need to *use* the result of an `async` function you need to await it first; but you don't, so no problem.